### PR TITLE
[sailfish-browser] Fix input region when in landscape

### PR DESCRIPTION
**Pull request by mikkoharju**

### CODE REVIEW

File: src/pages/BrowserPage.qml [hunk 1]
```diff
@@ -133,13 +133,21 @@ Page {
...
     InputRegion {
-        window: webView.chromeWindow
+        window: webView.portrait ? webView.chromeWindow : null
         y: webView.enabled && browserPage.active ? overlay.y : 0
         width: browserPage.width
         height: webView.enabled && browserPage.active ? browserPage.height - overlay.y : browserPage.height
     }
 
+    // Handle input region when landscape
+    InputRegion {
```
Comment (rainemak) [inline src/pages/BrowserPage.qml]: How about inverted landscape/portrait?

Would it be cleaner to move this handling to C++ and having a rotatingItem property.

File: src/pages/BrowserPage.qml [hunk 2]
```diff
@@ -141,12 +143,37 @@ Page {
...
+            var overlayVisibleHeight = browserPage.height - overlay.y
+
+            switch (orientation) {
+            case Orientation.None:
+            case Orientation.Portrait:
+                mask.y = overlay.y
+                // fallthrough
+            case Orientation.InvertedPortrait:
+                mask.height = overlayVisibleHeight
+                break
```
Comment (rainemak) [inline src/pages/BrowserPage.qml]: This doesn't work correctly. 

Input ends up being (x,y,w,h):  0 0 1536 2048. Thus, web content cannot be used.

File: src/pages/BrowserPage.qml [hunk 3]
```diff
@@ -141,12 +143,37 @@ Page {
         rotationHandler: browserPage
     }
 
+    function inputMaskForOrientation(orientation) {
```
Comment (rainemak) [inline src/pages/BrowserPage.qml]: Please move this around line 50 next to other function declarations.

File: src/pages/BrowserPage.qml (hunk 2 above)
Comment (mikkoharju) [inline src/pages/BrowserPage.qml]: Gah. InvertedPortrait vs PortraitInverted... no warnings, no nothing, js ftw

**Comment by rainemak** [comment]
Cheers @mikkoharju

**Pull request merged**